fcrepo-serialization/src/main/java/org/fcrepo/camel/serialization/SerializationRouter.java

```diff
@@ -75,8 +75,11 @@ public void configure() throws Exception {
         from("{{input.stream}}")
             .routeId("FcrepoSerialization")
             .process(new EventProcessor())
-            // this is a hard dependency on the jms module and should be reworked
-            .setHeader(FCREPO_IDENTIFIER).header("org.fcrepo.jms.identifier")
+            .process(exchange -> {
+                final String uri = exchange.getIn().getHeader(FCREPO_URI, "", String.class);
+                exchange.getIn().setHeader(FCREPO_IDENTIFIER, create(uri).getPath());
```

Comment:
Given:
* EventProcesser does not provide a value to `FCREPO_IDENTIFIER`
* The semantics of `FCREPO_IDENTIFIER` are slightly different here vs the previous revision

Was this change motivated by the desire to move toward different serialization paths (intentionally including /fcrepo/rest), or is it a side-effect of the fact that `FCREPO_IDENTIFIER` is no longer readily available; and it would be a lot more messy to produce values that were identical to past revisions (you'd need to configure a baseURI, then do then do a substring, etc)?   Would it make sense to remove `FCREPO_IDENTIFIER` from `FcrepoHeaders`?  I'm beginning to think that keeping it around is more confusing than not.